package.json

```
@@ -19,7 +19,7 @@
     "node": "14 || 16 || 17 || 18"
   },
   "dependencies": {
-    "@achrinza/event-pubsub": "5.0.6",
+    "@achrinza/event-pubsub": "https://github.com/achrinza/event-pubsub#4d0a3e3",
```

```suggestion
    "@achrinza/event-pubsub": "5.0.8",
```

TODO change when the change from https://github.com/achrinza/event-pubsub/pull/9 is released